pandas/tests/io/test_parquet.py

```diff
@@ -252,6 +301,10 @@ def test_duplicate_columns(self, fp):
 
     def test_unsupported(self, fp):
 
+        # bool with none
+        df = pd.DataFrame({'a': [True, None, False]})
```

Comment:
Yes
But it seems like a poor idea, when reading ints or bools (1-byte per value, maybe even the ints) to convert to Object just because there are some null values, more reasonable to use floats, which is definitely what pandas does with int values like [1, None, 3].
Would you say that the best encoding for bool-with-nulls is actually categorical with two possible values?